cfme/tests/services/test_dialog_regex_validation_in_catalog.py

```diff
@@ -65,3 +65,26 @@ def test_dialog_element_regex_validation(appliance, dialog_cat_item):
     wait_for(lambda: view.submit_button.disabled, timeout=7)
     view.fields(ele_name).fill("test_123")
     wait_for(lambda: not view.submit_button.disabled, timeout=7)
+
+
+@pytest.mark.manual
+@pytest.mark.tier(1)
+def test_dialog_text_area_element_regex_validation():
```

Comment:
Again why not automate this? the test case directly above is already using a fixture that tests a `Text Box` type element, why not just parametrize that fixture to include other element types, instead of a manual case for one specific type?